js/components/form.js

```diff
@@ -1,7 +1,7 @@
 import { allStates, allCountries, getMyData, hasUserData, translateHTML, translateText } from "../shared.js";
-import { addEventMonthSelection, addEventMonthConfirmationSelection, addEventUPSubmit, addEventCancerFollowUp, addEventChangeFocus, addEventAddressAutoComplete, addEventAdditionalEmail, addEventCheckCanText, addEventDisableCopyPaste } from "../event.js";
+import { addEventMonthSelection, addEventMonthConfirmationSelection, addEventUPSubmit, addEventCancerFollowUp, addEventChangeFocus, addEventAddressAutoComplete, addEventAdditionalEmail, addEventCheckCanText, addEventDisableCopyPaste, addEventFormerName, addMoreFormerName } from "../event.js";
 import cId from '../fieldToConceptIdMapping.js';
-import { suffixList, suffixToTextMapDropdown, suffixToTextMap } from "../settingsHelpers.js";
+import { suffixList, suffixToTextMapDropdown, suffixToTextMap, numberOfDefaultFormerNames } from "../settingsHelpers.js";
 
 export const renderUserProfile = async () => {
```

Comment:
state of birth can use `allStates` (shared.js) and be a dropdown selector to ensure no typos
country of birth can use `allCountries` (shared.js) and be a dropdown selector for the same reason.

This makes it more accurate when researchers quantify results over time.

There are examples of the allStates implementation in the existing code, and the allCountries implementation should basically be the same thing.

Reply:
This is confirmed by Rohan

![image](https://github.com/user-attachments/assets/fe190311-ede1-4e03-be97-13125cb59a18)

Reply:
sounds good, thanks